pandas/io/excel.py

```diff
@@ -373,6 +388,12 @@ def _parse_cell(cell_contents, cell_typ):
                 val = int(cell_contents)
                 if val == cell_contents:
                     cell_contents = val
+            elif true_values and cell_contents in true_values:
+                # GH13347
```

Comment:
you are going to need to make a set of these if they r not none

eg this will break with a scalar